### ISSUE TITLE
test ingame prevents updating of animations until used again

### ISSUE THREAD

**Issue by Xigzagamer** [tom5454/CustomPlayerModels#69]
when you use test ingame your animations are stuck as the last set you tested using the test ingame feature and don't update until you use it again. This happens even if you delete or create new animations and persists even after you close and reopen minecraft.

**Comment by tom5454** [comment]
The animations are encoded into the skin layer settings, and they are saved, by minecraft. (You can see it yourself if you activate a gesture or pose, and look in Options/Skin Customization).

**Comment by Xigzagamer** [comment]
I suppose I was a little unclear, basically the problem is when I use test in game, in the specific world I used to test it the model itself would be stuck as the last one I tested in that world. So it won't update even if I made a new export of my skin and applied it, or made additional changes since the last time I tested.

For example, I made a new piston texture and even after changing it, adding animations, etc. It would not update until I used the testing feature again in that world, however it would display fine on the server I play on.

Honestly as a feature it would be nice to keep the functionality of saving the skin to the world, but being tied to the testing feature means it is very easy to accidentally lock your skin in a world to the last one you tested without realizing it.

**Comment by tom5454** [comment]
Yes using test ingame didn't unload the testing model after exiting in some cases, this was fixed in 0.0.12a.
